scipy/optimize/tests/test_optimize.py

```diff
@@ -1145,6 +1145,17 @@ def test_l_bfgs_b_bounds(self):
         assert_(d['warnflag'] == 0, d['task'])
         assert_allclose(x, self.solution, atol=1e-6)
 
+    @pytest.mark.parametrize('bounds', [
+        ([(10, 1), (1, 10)]),
+        ([(1, 10), (10, 1)]),
+        ([(10, 1), (10, 1)])
+    ])
+    def test_l_bfgs_b_wrong_bounds(self, bounds):
+        with pytest.raises(ValueError, match='.*bounds.*'):
+            optimize.fmin_l_bfgs_b(self.fun, [0, -1],
```

Comment:
Please can you do the test using `optimize.minimize` using the `L-BFGS-B` method rather than `fmin_l_bfgs_b` function? 
i.e. in a similar fashion to `test_minimize_l_bfgs_b_bounds` a few lines below.